src/authors/scss/_body.scss

```diff
@@ -0,0 +1,68 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+body{
```

Comment:
NEVER style `tags` if it isn't required or the situation doesn't make it easier. Add proper classes and style them.